docs/guide/index.md

```diff
@@ -6,10 +6,10 @@ Many ongoing efforts are already happening in this space, from [dependency tree
 and much more.
 
 Our aim is to provide a space for contributions, ideas and knowledge sharing around the importance of performance across the ecosystem. Hopefully, with this collaboration, we can improve visibility of the impact of things like
-dependency choices. Join us at the [e18e Discord Server](https://chat.e18e.dev) to connect with other like-minded devs.
+dependency choices. Join us at the [e18e Discord server](https://chat.e18e.dev) to connect with other like-minded devs.
 
 Currently, there are three main areas of focus:
 
-- [cleanup](./cleanup.md) - cleaning up dependency trees and modernizing popular tools and libraries across the ecosystem.
-- [speedup](./speedup.md) - speeding up parts of the ecosystem many of us depend on.
-- [levelup](./levelup.md) - documenting and providing modern, lighter alternatives to established tools and libraries we all regularly use.
+- [Clean up](./cleanup.md): cleaning up dependency trees and modernizing popular tools and libraries across the ecosystem
```

Comment:
same as the others - these ones are names so can be left as-is (with the added colons)